valhalla/mjolnir/edgeinfobuilder.h

```diff
@@ -103,6 +103,20 @@ class EdgeInfoBuilder {
    */
   std::size_t SizeOf() const;
 
+  /**
+   * Check if the given name_info exists in this edge info.
+   * @param  offset  The offset of the name info.
+   * @return  Returns true if the name info exists, false otherwise.
+   */
+  bool has_name_info(uint32_t offset) const {
+    for (const auto& name : name_info_list_) {
```

Comment:
if adding landmarks turns out to be slow this is one thing we could speed up with an unordered_set of offsets as they are added and when they are deserialized